.travis.yml

```diff
@@ -1,44 +1,50 @@
 sudo: required
+language: generic
 
 branches:
   only:
     - master
 
+# Cache is irrelevant for linux/docker builds, but useful to speed up osx
 cache:
   directories:
     - $HOME/.pyenv
-    - googletest
-    - swig-3.0.10
+    - $HOME/googletest
```

Comment:
There's no caching of brew directories. It _should_ be faster to get deps via brew than the Travis cache